src/utils/durableUtils.ts

```diff
@@ -150,39 +154,46 @@ export namespace durableUtils {
     }
 
     async function installDotnetDependencies(context: IFunctionWizardContext): Promise<void> {
-        const packageNames: string[] = [];
+        const packages: { name: string; prerelease?: boolean }[] = [];
         const isDotnetIsolated: boolean = /Isolated/i.test(context.functionTemplate?.id ?? '');
 
         switch (context.newDurableStorageType) {
             case DurableBackend.Netherite:
                 isDotnetIsolated ?
-                    packageNames.push(dotnetIsolatedDfNetheritePackage) :
-                    packageNames.push(dotnetInProcDfNetheritePackage);
+                    packages.push({ name: dotnetIsolatedDfNetheritePackage }) :
+                    packages.push({ name: dotnetInProcDfNetheritePackage });
+                break;
+            case DurableBackend.DTS:
+                // Todo: Remove prerelease flag once this functionality is out of preview
```

Comment:
nit: why call the property prerelease when you call it preview everywhere else (including this comment)